charts/kubewarden-defaults/templates/allow-privileged-escalation-policy.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if .Values.bestPracticePolicies.enabled }}
+{{ if .Values.bestPracticePolicies.clusterWide }}
+apiVersion: policies.kubewarden.io/v1alpha2
+kind: ClusterAdmissionPolicy
+metadata:
+  name: {{ .Values.bestPracticePolicies.allowPrivilegeEscalationPolicy.name }}
+{{ include "allow-privileged-escalation-policy-spec" . }}
+{{ end }}
+{{- range $namespace := .Values.bestPracticePolicies.namespaces }}
+apiVersion: policies.kubewarden.io/v1alpha2
+kind: AdmissionPolicy
+metadata:
+  name: "{{ $.Values.bestPracticePolicies.allowPrivilegeEscalationPolicy.name }}-{{ $namespace }}"
```

Comment:
We can avoid the `namespace` repetition on the naming. Our namespaced controller will already add the namespace where needed to avoid collisions.

Reply:
I think you started your review before my lastest changes. I've already removed the suffix:

```console
head -10  templates/allow-privileged-escalation-policy.yaml
{{ if .Values.bestPracticePolicies.enabled }}
{{- range $namespace := .Values.bestPracticePolicies.namespaces }}
---
apiVersion: policies.kubewarden.io/v1alpha2
kind: AdmissionPolicy
metadata:
  name: {{ $.Values.bestPracticePolicies.allowPrivilegeEscalationPolicy.name }}
  namespace: {{ $namespace }}
spec:
  policyServer: {{ $.Values.policyServer.name }}
```